### ISSUE TITLE
Animation Trembling at METRO app

### ISSUE THREAD

**Issue by birbilis** [panthernet/GraphX#26]
at METRO app, there is lots of animation trembling when one changes algorithm (also the 2nd option does the same, 3d one doesn't) and presses the refresh button

**Comment by panthernet** [comment]
Yes i saw animation problems in METRO app but had no time to play with the logic. It seems that current implementation lacks sufficient update frequency when vertex coordinates changes. Also i see strange lable position tremble on zooom and was unable to find the reason. Thought it might be somehow related to layout rounding or coordinate value truncation.

**Comment by birbilis** [comment]
I think on Silverlight for example you only had integer positions you could specify on a Canvas, but I might remember wrong (maybe it was instead that actual calculated positions after transforms where truncated instead of rounded into pixel coords)
